dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -310,7 +325,9 @@ public final Response getContentTypes(@Context final HttpServletRequest request,
 		try {
 			response = this.paginationUtil.getPage(request, user, filter, page, perPage, orderBy, direction);
 		} catch (Exception e) {
-
+			if (ExceptionUtil.causedBy(e, DotSecurityException.class)) {
```

Comment:
it would be nice to have something like

ExceptionUtil.wrapIfCauseBy (e, DotSecurityException.class, ForbiddenException.class);